Source/NexusForever.WorldServer/Game/Entity/Entitlement.cs

```
@@ -22,6 +22,9 @@ public uint Amount
             get => amount;
             set
             {
+                if (value > Entry.MaxCount)
```

Would it be better to have an exception here rather than swallowing?

---

dis is not swallowing

---

It's taking a value that should be considered an exception and setting it back within a acceptable range.
It's hiding a potential problem, it should really throw an exception to fail early.